File: dev-docs/snippets/documents/list-pending-invitations.swift
import JsBaoClient

// List pending (deferred) email invitations scoped to a document. Swift
// returns an untyped `[[String: Any]]`.
func listPendingInvitations(client: JsBaoClient, documentId: String) async throws {
  // #region example
  let pending = try await client.documents.listPendingInvitations(
    documentId: documentId
  )
  // #endregion example
  _ = pending
}

File: dev-docs/snippets/documents/list-pending-invitations.ts
import type { JsBaoClient } from "js-bao-wss-client";

// List pending (deferred) email invitations scoped to a document.
export async function listPendingInvitations(
  client: JsBaoClient,
  documentId: string,
) {
  // #region example
  const pending = await client.documents.listPendingInvitations(documentId);
  // #endregion example
  return pending; // each: { email, permission, invitationId, createdAt, expiresAt }
}
